fails:Kernel#raise is a private method
fails:Kernel#raise aborts execution
fails:Kernel#raise raises RuntimeError if no exception class is given
fails:Kernel#raise raises a given Exception instance
fails:Kernel#raise raises a RuntimeError if string given
fails:Kernel#raise raises a TypeError when passed a non-Exception object
fails:Kernel#raise raises a TypeError when passed true
fails:Kernel#raise raises a TypeError when passed false
fails:Kernel#raise raises a TypeError when passed nil
fails:Kernel#raise re-raises the rescued exception
fails:Kernel#raise allows Exception, message, and backtrace parameters
fails:Kernel.raise needs to be reviewed for spec completeness
